k8s/manifests/kube-prometheus/monitoring.jsonnet

```diff
@@ -27,7 +28,8 @@ local grafanaDashboards =
   postgres.grafanaDashboards +
   redis.grafanaDashboards +
   ingressNginx.grafanaDashboards +
-  simpleServer.grafanaDashboards;
+  simpleServer.grafanaDashboards +
+  dhis2Server.grafanaDashboards;
```

Comment:
Suggestion: We may not want to show the dhis2 dashboards in all environments. 
You can add a config value per env to enable/disable them.